python/src/aac/io/parser/_parse_source.py

```diff
@@ -104,7 +104,7 @@ def is_token_between_locations(token, inclusive_line_start: int, inclusive_line_
 
         end_of_file_offset = 1 if isinstance(end_doc_token, StreamEndToken) else 0
 
-        yaml_text = linesep.join(model_content.splitlines()[content_start_line:content_end_line + end_of_file_offset])
+        yaml_text = linesep.join(model_content.splitlines()[content_start_line: content_end_line + end_of_file_offset])
```

Comment:
I've had this kind of change cause linter warnings. How were you able to get around that?

Reply:
I... I don't know. I wonder if a rule was disabled.

Reply:
No, I figured it out (although it's a bit weird). `flake8` is configured to allow space after a colon (which makes sense for - e.g. `lambda: None`, etc) but not before one and the [E203](https://www.flake8rules.com/rules/E203.html) error - i.e. "whitespace before ':'" - is what I was seeing.